pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -19,6 +20,17 @@ let
     packageOverrides = self: super: {
       django = super.django_4;
 
+      # use paperless-ngx version of django-q
+      # see https://github.com/paperless-ngx/paperless-ngx/pull/1014
+      django-q = super.django-q.overridePythonAttrs (oldAttrs: rec {
+        src = fetchFromGitHub {
```

Comment:
Should maybe also change the `version`? Not sure what to...

Reply:
I would leave it, as is. The fork didn't change the version either, and paperless-ngx is a leaflet package, so it shouldn't conflict with any downstream packages